view/frontend/templates/button_product_page.phtml

```diff
@@ -27,7 +27,13 @@ if (!$block->isBoltProductPage()) return;
 
 if (!$block->isSupportableType()) return;
 ?>
-    <div class="bolt-product-checkout-button bolt-multi-step-checkout <?=$block->getAdditionalCheckoutButtonClass();?>" style="<?=$block->getButtonCssStyles(); ?>"></div>
+    <?php if(!$block->getIsInstantCheckoutButton()){?>
+        <div class="bolt-product-checkout-button bolt-multi-step-checkout <?=$block->getAdditionalCheckoutButtonClass();?>" style="<?=$block->getButtonCssStyles(); ?>"></div>
+    <?php } else { ?>
+        <div data-tid="instant-bolt-checkout-button" style="<?=$block->getButtonCssStyles(); ?>">
```

Comment:
PPC button (Buy Now) used to have a different class than the regular multi-step checkout one, `bolt-product-checkout-button` vs. `bolt-checkout-button`. With the current V2 button implementation there's no difference. Looks like an error. If it's not, please add an explanation how this work on PCC.

Reply:
You are correct it was not working as expected on that page. I spoke to Pranay and V2 is not ready for product page.